RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -108,21 +109,16 @@ class VoiceBroadcastPlaybackViewModel: VoiceBroadcastPlaybackViewModelType, Voic
         displayLink.isPaused = false
         isActuallyPaused = false
         
-        if voiceBroadcastAggregator.isStarted == false {
-            // Start the streaming by fetching broadcast chunks
-            // The audio player will automatically start the playback on incoming chunks
-            MXLog.debug("[VoiceBroadcastPlaybackViewModel] play: Start streaming")
-            state.playbackState = .buffering
-            voiceBroadcastAggregator.start()
-        } else if let audioPlayer = audioPlayer {
+        if let audioPlayer = audioPlayer {
             MXLog.debug("[VoiceBroadcastPlaybackViewModel] play: resume")
             audioPlayer.play()
-        } else {
+        } else if voiceBroadcastAggregator.launchState == .loaded {
+            state.playbackState = .buffering
             let chunks = voiceBroadcastAggregator.voiceBroadcast.chunks
             MXLog.debug("[VoiceBroadcastPlaybackViewModel] play: restart from the beginning: \(chunks.count) chunks")
             
             // Reinject all the chunks we already have and play them
-            voiceBroadcastChunkQueue.append(contentsOf: chunks)
+            voiceBroadcastChunkQueue = Array(chunks)
             processPendingVoiceBroadcastChunks()
```

Comment:
replace "processPendingVoiceBroadcastChunks" with "handleVoiceBroadcastChunksProcessing" to handle live VB correctly

Reply:
Fixed in last commit.